lib/postgrex/protocol.ex

```diff
@@ -343,12 +346,19 @@ defmodule Postgrex.Protocol do
   end
 
   def handle_prepare(%Query{name: ""} = query, opts, s) do
-    prepare = Keyword.get(opts, :postgrex_prepare, false)
-    status = new_status(opts, prepare: prepare)
+    prepare? = Keyword.get(opts, :postgrex_prepare, false)
+    status = new_status(opts, prepare: prepare?)
 
-    case prepare do
-      true -> parse_describe_close(s, status, query)
-      false -> parse_describe_flush(s, status, query)
```

Comment:
Can you please avoid stale changes on the PR? Please keep the variable as `prepare` and using `case` instead of `if`. This makes it easier to review and debug history later. Thank you :)